### PR TITLE
Documentation/166 update readme

### PR DESCRIPTION
**Pull request by daxicek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
-flutter packages pub run build_runner build
-```
+* [Join our Community](https://discord.com/invite/aKhjuwZmdP)
+* [Join our Telegram group](https://t.me/simplioofficial)
+* [Follow us on Twitter](https://twitter.com/simplioofficial)
+* [Send us Email](mailto://support@simplio.io)
+* [Visit Web](https://simplio.io)
 
-## Available parameters
-* --dart-define=API_URL=PLACEHOLDER_URL
```
Comment (jvrastil) [inline README.md:22]: why did you remove this section? It was quite important so we can track all possible options...

Comment (daxicek) [reply README.md:22]: Technical documentation will be part of different file.

Comment (ciripel) [reply README.md:22]: Then create that file also in this PR and push together. We would not like to loose this starting app information

Comment (ciripel) [reply README.md:22]: Also create a small README for crypto_assets!

**Review by ciripel — CHANGES_REQUESTED** [review]
Create file where you will have technical documentation how to start the app with ENV VARIABLES also in this PR and push together. We would not like to loose this starting app information
Also create a small README for crypto_assets!

**Comment by daxicek** [comment]
Added small technical documentation

**Review by jvrastil — APPROVED** [review]

**Review by ciripel — APPROVED** [review]
